cmake/config.cmake

```diff
@@ -118,7 +125,7 @@ set(USE_MICRO_STANDALONE_RUNTIME OFF)
 # - ON: enable llvm with cmake's find search
 # - OFF: disable llvm
 # - /path/to/llvm-config: enable specific LLVM when multiple llvm-dev is available.
-set(USE_LLVM OFF)
+set(USE_LLVM ON)
```

Comment:
We probably don't need this change. In stead, we usually set it up in the local `build/config.cmake`